src/rgen/Microsoft.Macios.Generator/DataModel/Method.Generator.cs

```diff
@@ -35,6 +35,12 @@ readonly partial struct Method {
 	/// </summary>
 	public bool MarshalNativeExceptions => ExportMethodData.Flags.HasFlag (ObjCBindings.Method.MarshalNativeExceptions);
 
+	/// <summary>
+	/// True if the generator should not use a NSString for marshalling.
+	/// </summary>
+	public bool UsePlainString
+		=> ExportMethodData.Flags.HasFlag (ObjCBindings.Method.PlainString);
```

Comment:
HasFlag used to have some perf implication, I think they fixed that (?)
alternative is `=> value & flag == flag`

Reply:
Yes, they did fix it.